deps/shims/Deno.node.ts

```diff
@@ -18,6 +19,9 @@ let _deno = {
     },
     toObject: () => ({ ...process.env }),
   },
+  readFileSync: (path: string | URL): Uint8Array =>
+    Buffer.from(fs.readFileSync(path, "binary"), "binary"),
```

Comment:
`fs.readFileSync` already returns a `Buffer`, so there's no need to use `Buffer.from` here.